src/pages/resultsView/enrichments/AlterationEnrichmentsContainer.tsx

```diff
@@ -395,10 +399,65 @@ export default class AlterationEnrichmentContainer extends React.Component<
         );
     }
 
+    @computed private get geneBarplotYAxislabel() {
+        if (
+            this.isAnyMutationTypeSelected &&
+            !this.isAnyFusionTypeSelected &&
+            !this.isAnyCnaTypeSelected
+        )
+            return 'Mutation frequency';
+        if (
+            !this.isAnyMutationTypeSelected &&
+            this.isAnyFusionTypeSelected &&
+            this.isAnyCnaTypeSelected
+        )
+            return 'Fusion event frequency';
+        if (
+            !this.isAnyMutationTypeSelected &&
+            !this.isAnyFusionTypeSelected &&
+            !this.isAnyCnaTypeSelected
+        )
+            return 'Copy-number alteration frequency';
+        return 'Alteration event frequency';
+    }
+
+    @computed get isAnyMutationTypeSelected() {
+        if (!this.props.comparisonStore) return false;
+        const typeMap = this.props.comparisonStore!
+            .selectedMutationEnrichmentEventTypes;
+        return _.find(
```

Comment:
since this is meant to be boolean, you should use _.some instead of find.   As it is, TS will infer the type incorrectly

Reply:
I changed this and other occurrence and other that I made this way.